src/platform/monitoring/DowntimeNotification/actions/index.js

```diff
@@ -50,18 +55,42 @@ export function dismissDowntimeWarning(appTitle) {
 }
 
 export function getScheduledDowntime() {
-  return async dispatch => {
+  return async (dispatch, state) => {
     dispatch({ type: RETRIEVE_SCHEDULED_DOWNTIME });
     let data;
+
+    // create global downtime data if feature toggle is enabled and if
+    // current time is in downtime window
+    // default to empty array
+    const { downtimeStart, downtimeEnd } = scheduledDowntimeWindow;
+    const globalDowntimeData =
+      (toggleValues(state).vaGlobalDowntimeNotification &&
+        moment().isAfter(downtimeStart) &&
+        moment().isBefore(downtimeEnd) &&
+        createGlobalMaintenanceWindow({
+          startTime: downtimeStart,
+          endTime: downtimeEnd,
+        })) ||
+      [];
     try {
       const response = await apiRequest('/maintenance_windows/');
-      data = response.data;
+      if (!response.ok) {
+        dispatch({
+          type: ERROR_SCHEDULE_DOWNTIME,
+          data: globalDowntimeData,
+        });
```

Comment:
Just some context - the error-handler was left empty before because if there was an error retrieving maintenance windows from the API, it would be too bold to make any assumptions off of that that would affect an app or the UI in general.

Reply:
I think we'll have some discussion about this soon because i think if the status api is down, then it's likely the whole site is down.